css/properties.json

```diff
@@ -4220,6 +4220,50 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/contain"
   },
+  "contain-intrinsic-size": {
+    "syntax": "[ none | <length> | auto <length> ]{1,2}",
+    "media": "visual",
+    "inherited": false,
+    "animationType": [
+      "contain-intrinsic-width",
+      "contain-intrinsic-height"
```

Comment:
NOte, I just copied the pattern of another case and listed all the longhand properties for all of these because the spec said "see individual properties".